easybuild/easyblocks/w/wps.py

```diff
@@ -110,14 +110,14 @@ def configure_step(self):
         # libpng dependency check
         libpng = get_software_root('libpng')
         zlib = get_software_root('zlib')
+        libpnglib = ""
+        libpnginc = ""
```

Comment:
@mboisson I'd throw this in the (original) `else` part, with a clear comment like

```
# define these as empty, assume that libpng will be available via OS (e.g. due to --filter-deps=libpng)
```